types/react-intl/index.d.ts

```diff
@@ -188,6 +188,7 @@ declare namespace ReactIntl {
             locale?: string;
             formats?: any;
             messages?: any;
+            textComponent?: any;
```

Comment:
Sorry for the late reply. According to the react-intl source code:

```
export const intlConfigPropTypes = {
    locale       : string,
    formats      : object,
    messages     : object,
    textComponent: any,

    defaultLocale : string,
    defaultFormats: object,
};
```
from https://github.com/yahoo/react-intl/blob/master/src/types.js, tells me that `textComponent` can be of type `any`. From the use cases I know, it can either take `string` (ex> `span`, `div`) or `node` (ex> `Text` from react native). I left it as `any` since the source code in react-intl specifies the type as `any` for `textComponent` prop.